src/clims/models/assignment.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import
+
+from django.db import models
+from sentry.db.models import (FlexibleForeignKey, sane_repr)
+from jsonfield import JSONField
+
+
+class SubstanceAssignment(models.Model):
```

Comment:
I believe you might be thinking about my comment to the class that this only has to do with human interactions? I think that it might be indicated by that these are "assignments" to a workflow that they must be made by a user, but I wanted to be clear on that this model doesn't have information on what happened in the workflow after that happened. I might have to revisit that documentation to clarify it.

So to clarify, there is no assignment made by an entity that's not a user, so I guess it might be unnecessary to say that it's a user because it's implied.

Reply:
:+1: